modules/ROOT/pages/rate-limiting.adoc

```diff
@@ -0,0 +1,219 @@
+
+* N servers for the same API
+* Single backend to which all of the proxies connect
+* Possible load balancer in front of the proxies
+
+In such a scenario, you do not need a cluster. If q is the maximum capacity of the backend, then configure the policy as follows:  where  is a small number, lower than the backend capacity.
+
+If there are no load balancers in your environment, the cluster mode is recommended over a standalone, because you cannot configure beforehand how much traffic each node will handle. The Rate Limiting policy is designed to work both on a perfectly balanced workload, or on a completely uneven one. The backend does not receive any extra requests and thus does not exceed the maximum capacity that it can handle.
+
+=== Window Sizes in Clusters
+
+In clusters, the nodes must share information for consistency across the cluster. The sharing process adds latency that must be taken into account when reviewing performance.
+
+In a worst case scenario, the number of penalized requests with latency due to cluster consistency is constant and independent from the actual size of the configured quota. Consequently, the smaller the window, the greater the percentage of potentially delayed requests. 
+
+Therefore, MuleSoft recommends that you set up only window sizes greater than one minute in Rate Limiting and Rate Limiting SLA policy configurations for the cluster scenario.
+
+=== Persistence
+
+You can configure Rate Limiting policy to use big windows sizes, such asdays, months, and years. For example, suppose as a client you want to allow your user ‘X’ to consume 1 million requests per year. You cannot predict whether the node will be up the entire period or will need maintenance, which may result in restarting the Mule runtime engine. 
+
+The algorithm has been running for several months, so the client will lose critical information. Persistence solves this problem by periodically saving the current policy state. In case of a redeployment or a restart, the algorithms are recreated from the last known persisted state or started from a clean state.
+
+Although persistence is enabled by default, you can turn it off by setting the following property to false:
+
+`throttling.persistence_enabled`
+
+You can also tweak the persistence frequency rate, which has a default of 10 seconds:
+throttling.persistent_data_update_freq
+
+[Note]
+---
+Persistence is not available on CloudHub.
+---
+
+== Configuring Policy Parameters
+
+When you apply the Rate Limiting policy to your API from the UI, you can configure the following parameters:
+
+[%header%autowidth.spread,cols="a,a,a"]
+|===
+|Parameter | Description | Example
+|Identifier | Specifies a selector key using a Dataweave or regular string | `#[attributes.method]` This creates one group for each available method in HTTP, and for example, the policy will rate limit GET requests independently from POST requests.
+|# of Reqs | The quota available per window | A positive number, for example: 3.
```

Comment:
"A positive number, for example: 3."     >
    A positive number, for example, 3
or
   A positive number like 3
or
   A positive number

...because everyone knows what a positive number is. And perhaps you need "positive whole number"?

Reply:
Done! Thanks!